app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue

```diff
@@ -117,6 +117,8 @@ export default {
   color: var(--r-500);
 }
 .input-group {
```

Comment:
@iamsivin I don't think this is a correct fix. This issue came from https://github.com/chatwoot/chatwoot/pull/5103/files#diff-fe5d6ca412a0ed9343c2a796a0de32177375af5653e9ac0e2252d6cc8c004d00R67

Let us revert this change or update it properly. 

cc: @muhsin-k

Reply:
Yeah will check it and update

Reply:
@pranavrajs Updated https://github.com/chatwoot/chatwoot/pull/5183/commits/8503e1edd4f3a61ed636743e88579e15c13a0996 . Can you take a look?